LOAN/LoanTypes/LoanProducts.rdf

```diff
@@ -52,35 +52,66 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;hasPreconditions"/>
-				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/LoanPrecedentConditions"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;Collateral"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;isTemplateFor"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;LoanContract"/>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;hasPreconditions"/>
+				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/LoanPrecedentConditions"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;specifies"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-col;LoanCollateral"/>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;isTemplateFor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;AutoLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">auto loan purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;BridgingLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">bridging loan purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;ConstructionLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">construction loan purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;CreditCardReceivablePurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">credit card receivable purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;HomeEquityLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">home equity loan purpose</rdfs:label>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanProductDisclosureRight">
 		<rdfs:subClassOf rdf:resource="&fibo-loan-loan-reg;ProductDisclosureRight"/>
 		<rdfs:label xml:lang="en">loan product disclosure right</rdfs:label>
 		<skos:definition xml:lang="en">Right to fair representation of the facts about Loan products before entrering into agreements for same.</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-gao-obj;Objective"/>
```

Comment:
Generally a loan purpose is more general than the definition of Objective would allow.

Reply:
ok - well this whole thing needs more work ...